### PR TITLE
[BugFix] Fix NullPointerException log when load fail

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

when load fail in prepare step, there will be no failed tablets. so that it will log a NullPointerException in PostAbort() which handle the failed tablets. the consequence is ok but log is unnecessary. 

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
Can't set status; build succeeded.

**Comment by meegoo** [comment]
run all

**Comment by meegoo** [comment]
run all

File: fe/fe-core/src/main/java/com/starrocks/transaction/OlapTableTxnStateListener.java [hunk 1]
```diff
@@ -277,7 +278,7 @@ public void postAbort(TransactionState txnState, List<TabletFailInfo> failedTabl
                     }
                 }
             } catch (Exception e) {
-                LOG.warn(e);
+                LOG.warn(DebugUtil.getStackTrace(e));
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/transaction/OlapTableTxnStateListener.java:281]:
```suggestion
                LOG.warn("Fail to execute postAbort", e);
```
In this way, the stack of `e` will be printed

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 1 / 3 (33.33%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/transaction/OlapTableTxnStateListener.java|0|2|00.00%|[265, 280]|
|:large_blue_circle:|com/starrocks/transaction/GlobalTransactionMgr.java|1|1|100.00%|[]|

**Review by imay — APPROVED** [review]

**Comment by imay** [comment]
run starrocks_admit_test

**Review by Astralidea — APPROVED** [review]

**Comment by meegoo** [comment]
@mergify backport branch-2.5

**Comment by meegoo** [comment]
@mergify backport branch-3.0

**Pull request merged**